src/main/resources/migrations/00005_drop_table_catalogues.sql

```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:drop_table_catalogues
+ALTER TABLE products DROP  CONSTRAINT fk_products_catalogues;
+
+DROP TABLE catalogues;
+
```

Comment:
Don't we need rollback on this and 00006?

Reply:
We can't really roll back the drop of a table or a column. Either the table (or column) been dropped, in which case we lost the existing data or the drop command hasn't been executed, in which case the table (or column) is retained with the existing data.

Reply:
Thanks :)